docs/pages/api-docs/data-grid.md

```diff
@@ -62,6 +62,7 @@ import { DataGrid } from '@material-ui/data-grid';
 | <span class="prop-name">showCellRightBorder</span> | <span class="prop-type">boolean</span> | false | If `true`, the right border of the cells are displayed. |
 | <span class="prop-name">showColumnRightBorder</span> | <span class="prop-type">boolean</span> | false | If `true`, the right border of the column headers are displayed. |
 | <span class="prop-name">showToolbar</span> | <span class="prop-type">boolean</span> | false | If `true`, the default toolbar component is shown. |
+| <span class="prop-name">selectionModel</span> | <span class="prop-type">SelectionModel</span> |   | Set the selection model of the grid. |
```

Comment:
We aim to sort the props by alphabetical order. It's done automatically by the generation script, but since we don't use yet.